docs/install_guides/uninstalling-openvino.md

```diff
@@ -2,47 +2,66 @@
-   ```sh
-   sudo ./uninstall_GUI.sh
-   ```
-2. Follow the uninstallation wizard instructions.
-
-
-### Uninstall with Command Line (Interactive Mode)
-1. Run the uninstallation script from `<INSTALL_DIR>/openvino_toolkit_uninstaller`:
-   ```sh
-   sudo ./uninstall.sh
-   ```
-2. Follow the instructions on your screen.
-4. When uninstallation is complete, press **Enter**.
-
-### Uninstall with Command Line (Silent Mode)
-1. Run the following command from `<INSTALL_DIR>/openvino_toolkit_uninstaller`:
-   ```sh
-   sudo ./uninstall.sh -s
-   ```
-2. Intel® Distribution of OpenVINO™ Toolkit is now uninstalled from your system.
-
-## <a name="uninstall"></a>Uninstall the Toolkit in Windows
-
-Follow the steps below to uninstall the Intel® Distribution of OpenVINO™ Toolkit from your system:
-1. Choose the **Apps & Features** option from the Windows* Settings app.
-2. From the list of installed applications, select the Intel® Distribution of OpenVINO™ Toolkit and click **Uninstall**.
-3. Follow the uninstallation wizard instructions.
-4. When uninstallation is complete, click **Finish**. 
-
-## <a name="uninstall"></a>Uninstall the Toolkit in macOS
-
-Follow the steps below to uninstall the Intel® Distribution of OpenVINO™ Toolkit from your system:
-
-1. Run the uninstall application from the installation directory (by default, `/opt/intel/openvino_2021`):
-   ```sh
-   open openvino_toolkit_uninstaller.app
-   ```
-2. Follow the uninstallation wizard instructions.
-3. When uninstallation is complete, click **Finish**. 
+##There are 2 ways of uninstalling OpenVINO:
+
+### 1. Bootstrapper file
+
+@sphinxdirective
+.. tab:: Windows
+
+  Go to the ``Downloads`` folder and double-click ``w_openvino_toolkit_p_<version>.exe``. Follow the wizard instructions.
+
+.. tab:: Linux
+
+  If you want to use graphical user interface (GUI) installation wizard, run the script without any parameters:
+  
+  .. code-block:: sh
+  
+    ./l_openvino_toolkit_p_<version>.sh
+
+  Follow the wizard instructions.
+
+  Otherwise, you can add parameters `-a` for additional arguments and `--cli` to run installation in command line (CLI):
+  
+  .. code-block:: sh
+    
+    ./l_openvino_toolkit_p_<version>.sh -a --cli
+
+  Follow the wizard. Select **Remove** option when presented.
+
+.. tab:: macOS
+
+  Go to the ``Downloads`` folder and double-click ``m_openvino_toolkit_p_<version>.dmg``. Follow the wizard instructions.
+
+@endsphinxdirective
+
+### 2. System specific
+
+@sphinxdirective
+.. tab:: Windows
+
+  1. Choose the **Apps & Features** option from the Windows* Settings app.
+  2. From the list of installed applications, select the Intel® Distribution of OpenVINO™ Toolkit and click **Uninstall**.
+  3. Follow the uninstallation wizard instructions.
+
+.. tab:: Linux & macOS
+
+  1. Run the installer file from the installation directory:
+   
+  .. code-block:: sh
+  
+    ./<INSTALL_DIR>/intaller/installer
```

Comment:
for macOS it should be: `open /opt/intel/openvino/installer/installer.app`
for Linux: `./<installdir>/openvino/installer/installer`,  for example `/home/<username>/intel/openvino/installer/installer` for user mode installation 
or `/opt/intel/openvino/installer/installer` in a case of administrative installation.

Reply:
I've split macos and linux to different tabs and amended the path for linux. However, I would keep ./<installdir>/openvino/installer/installer for linux without further examples to keep it as generic as possible.

Reply:
strictly speaking, `/opt/intel/openvino/` is not product installation directory, but installation directory of installer itself. 
IMO refereeing `./<INSTALL_DIR>/intaller/installer` bring confusion, since 
<install_dir> is /opt/intel/ or /home/<username>/intel/
<product_dir>  is /opt/intel/openvino_<version> or /home/<username>/intel/openvino_<version>
<installer_dir> is `/home/<username>/intel/openvino/installer/` or  `/opt/intel/openvino/installer`